src/locale/languages.ts

```diff
@@ -31,6 +32,7 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.hi, name: 'हिंदी'},
   {code2: AppLanguage.ja, name: '日本語'},
   {code2: AppLanguage.ko, name: '한국어'},
+  {code2: AppLanguage.pt, name: 'Portuguese (BR)'},
```

Comment:
I added this clarification @maisondasilva hope this is okay!

Reply:
Yes, yes, it is possible to add Brazilian Portuguese to the file languages.ts

https://github.com/bluesky-social/social-app/blob/032a5bd8a744d5096938301ac25b709a50a4db1d/src/locale/languages.ts#L429